skglm/penalties/block_separable.py

```diff
@@ -382,3 +382,109 @@ def generalized_support(self, w):
                 gsupp[g] = True
 
         return gsupp
+
+
+class WeightedSparseGroupL2(BasePenalty):
```

Comment:
@Badr-MOUFAD @QB3 need a better name (sparse could be from any other penalty). Any idea?

Reply:
`WeightedL1GroupL2` ?